guides/doc-Deploying_Project_on_AWS/topics/AWS-registering-hosts.adoc

```diff
@@ -0,0 +1,10 @@
+[[aws-registering-hosts]]
+= Registering Hosts to {Project}
+
+ifdef::foreman-el,katello[]
+This procedure applies only to users of the Katello plug-in.
+endif::[]
```

Comment:
I think we should drop this and instead only show the file for Katello, orcharhino, and Satellite.

Reply:
I'm confused.
Registering Hosts is enabled for foreman-el and foreman-deb as well. Isn't it still possible to register hosts even without Katello, only without using activation keys?

Reply:
I think we should hide the chapter altogether if it doesn't apply, rather than showing content that users shouldn't read. The "this only applies to Katello users" part is something we can deal with in asciidoc.

Reply:
Actually, if hosts can be registered regardless of Katello (with subtle differences), then I think we can do the contrary and include this chapter for all builds, because the linked chapter Registering Hosts in Managing Hosts covers registration for all builds on a subtler level. And remove the sentence in the condition as well.